src/interface_py/Makefile

```diff
@@ -70,6 +70,12 @@ install:
 #	python3 setup.py sdist bdist_wheel
 
 clean:
+	rm -f h2o4gpu/BUILD_INFO.txt
+	rm -f h2o4gpu/dist*
+    # A bit hacky b/c of how we are merging __init__.py with other frameworks
+    # This *is* required
+	rm -f h2o4gpu/__init__.py
```

Comment:
I wonder if there is a better solution? I think its fine for now, but just curious.

Reply:
In works (as in jenkins list issue and as separate issue) is to pull out sklearn completely.  That will fix build times, this, etc.